addons/project/tests/test_project_update.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields
+from odoo.addons.mail.tests.common import mail_new_test_user
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+from odoo.exceptions import AccessError
+from odoo.tests import HttpCase, tagged
+from odoo.tests.common import Form, users
```

Comment:
Based on the [guidelines](https://www.odoo.com/documentation/master/reference/guidelines.html#imports), we should do it like this:
```suggestion
from odoo import fields
from odoo.exceptions import AccessError
from odoo.tests import HttpCase, tagged
from odoo.tests.common import Form, users

from odoo.addons.mail.tests.common import mail_new_test_user
from odoo.addons.project.tests.test_project_base import TestProjectCommon
```